validate/validate.go

```diff
@@ -219,6 +219,11 @@ func (v *Validator) CheckRoot() (errs error) {
 	}
 
 	if v.platform == "windows" {
+		if !strings.HasPrefix(v.spec.Root.Path, `\\?\`) {
```

Comment:
We can be stricter than this.  The spec links [here][1], whivh would be a regex like `[\][\][?][\]Volume[
{][a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}[}][\]?"`.  The Windows docs say "GUID", I think they mean "[UUID][2]", based on their `\\?\Volume{26a21bda-a627-11d7-9931-806e6f6e6963}\` example.

Also, unit tests :).

[1]: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365248(v=vs.85).aspx
[2]: https://tools.ietf.org/html/rfc4122#section-3

Reply:
updated, PTAL.